statsmodels/discrete/tests/test_discrete.py

```diff
@@ -2390,3 +2390,22 @@ def test_cov_confint_pandas():
     assert_index_equal(ci.index, cov.index)
     assert_index_equal(cov.index, cov.columns)
     assert isinstance(ci.index, pd.MultiIndex)
+
+
+def test_t_test():
```

Comment:
Brief comment about the motivation for the test

Reply:
Added issue